src/lib/handle-result.ts

```diff
@@ -92,11 +106,30 @@ export function createSection(entry?: TestNameEntry): SectionSummary {
 }
 
 export function createUpdateFunctionSummary(
+    functionLeaders: FunctionLeaders,
     functionMap: FunctionMap,
     functions: FunctionEntry[],
-    entry: FunctionLocationEntry | FunctionExecutionEntry
+    entry: FunctionAliasEntry | FunctionLocationEntry | FunctionExecutionEntry
 ): void {
     const functionSummary = functionMap.get(entry.name)
+    if (entry.variant === Variant.FunctionAlias) {
+        // biome-ignore lint/style/noNonNullAssertion: Leader always comes before aliases of a function
+        const functionLeader = functionLeaders.get(entry.index)!
+        if (functionSummary === undefined) {
+            const summary = {
+                hit: entry.hit,
+                line: functionLeaders.get(entry.index)?.lineStart || 0,
```

Comment:
Since `0 || 0` will use the _else_ branch even though there is an actual value, consider using the [`??`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) operator to use `0` only when the value is absent (i.e.: undefined).

```suggestion
                line: functionLeaders.get(entry.index)?.lineStart ?? 0,
```